package.json

```diff
@@ -4,17 +4,19 @@
   "description": "Twilio Video Room Monitor Javascript Library",
   "main": "dist/node/index.js",
   "types": "dist/node/index.d.ts",
+  "regular": "dist/browser/twilio-video-room-monitor.js",
+  "minified": "dist/browser/twilio-video-room-monitor.min.js",
   "scripts": {
     "start": "tsc --watch",
     "test": "cross-env TZ=utc jest",
     "test:ci": "cross-env TZ=utc jest --ci --runInBand --reporters=default --reporters=jest-junit --coverage --silent",
     "lint": "eslint src",
-    "parcel:watch": "cross-env PARCEL_TARGET=browser parcel src/index.tsx --no-hmr --out-dir dist/browser --out-file twilio-video-room-monitor.js",
+    "parcel:watch": "cross-env PARCEL_TARGET=browser parcel src/index.tsx --no-hmr --dist-dir dist/browser --target regular",
     "prebuild": "rimraf dist/",
     "build": "npm run build:node && npm run build:browser && npm run build:browser:min",
     "build:node": "tsc",
-    "build:browser": "cross-env PARCEL_TARGET=browser parcel build src/index.tsx --no-source-maps --out-dir dist/browser --out-file twilio-video-room-monitor.js --target browser --no-minify",
-    "build:browser:min": "cross-env PARCEL_TARGET=browser parcel build src/index.tsx --no-source-maps --out-dir dist/browser --out-file twilio-video-room-monitor.min.js --target browser",
+    "build:browser": "cross-env PARCEL_TARGET=browser parcel build src/index.tsx --no-source-maps --target regular",
+    "build:browser:min": "cross-env PARCEL_TARGET=browser parcel build src/index.tsx --no-source-maps --target minified",
```

Comment:
One thing about these scripts, is that some options are CLI flags, and other options are in package.json. For clarity, let's just move them all to package.json. 

So I think these scripts can be:
```
    "build:browser": "cross-env PARCEL_TARGET=browser parcel build --target browser",
    "build:browser:min": "cross-env PARCEL_TARGET=browser parcel build --target minified",
```

And the package.json config can be:
```
  "targets": {
    "regular": {
      "source": "src/index.tsx",
      "distDir": "dist/browser",
      "optimize": false,
      "sourceMap": false,
      "scopeHoist": false
    },
    "minified": {
      "source": "src/index.tsx",
      "distDir": "dist/browser",
      "optimize": true,
      "sourceMap": false,
      "scopeHoist": true
    }
  }
```

Note, this also disables scope hoisting for the non-minified target.